service/src/main/java/org/cbioportal/genome_nexus/service/enricher/SignalAnnotationEnricher.java

```diff
@@ -0,0 +1,31 @@
+package org.cbioportal.genome_nexus.service.enricher;
+
+import org.cbioportal.genome_nexus.model.SignalAnnotation;
+import org.cbioportal.genome_nexus.model.SignalMutation;
+import org.cbioportal.genome_nexus.model.VariantAnnotation;
+import org.cbioportal.genome_nexus.service.AnnotationEnricher;
+import org.cbioportal.genome_nexus.service.SignalMutationService;
+
+import java.util.List;
+
+public class SignalAnnotationEnricher implements AnnotationEnricher
+{
+    private final SignalMutationService signalMutationService;
+
+    public SignalAnnotationEnricher(SignalMutationService signalMutationService) {
+        this.signalMutationService = signalMutationService;
+    }
+
+    @Override
+    public void enrich(VariantAnnotation annotation) {
+        SignalAnnotation signalAnnotation = new SignalAnnotation();
+        String hgvsg = annotation.getHgvsg();
```

Comment:
Always using `hgvgs` even if the actual query is originated from a genomic location query. It might be better to use the original query parameters if available, but we are only passing the `annotation` as a parameter here. May need to improve the enricher interface for that.

Reply:
agreed, might be good to improve the enricher interface

Reply:
maybe good to open a separate issue?

Reply:
open a new issue #411